cobbler/tftpgen.py

```diff
@@ -456,29 +457,30 @@ def get_submenus(self, menu, metadata: dict, arch: enums.Archs):
                     else:
                         nested_menu_items[boot_loader] = temp_items[boot_loader]
 
-            if "ipxe" in temp_items:
-                if "ipxe" not in menu_labels:
-                    menu_labels["ipxe"] = []
-                display_name = (
-                    child.display_name
-                    if child.display_name and child.display_name != ""
-                    else child.name
-                )
-                menu_labels["ipxe"].append(
-                    {"name": child.name, "display_name": display_name}
-                )
+                if boot_loader not in menu_labels:
+                    menu_labels[boot_loader] = []
+
+                if "ipxe" in temp_items:
+                    display_name = (
+                        child.display_name
+                        if child.display_name and child.display_name != ""
+                        else child.name
+                    )
+                    menu_labels[boot_loader].append(
+                        {"name": child.name, "display_name": display_name}
+                    )
 
         for boot_loader in boot_loaders:
             if (
                 boot_loader in nested_menu_items
-                and nested_menu_items[boot_loader] != ""
+                and nested_menu_items[boot_loader][-2:] == "\n\n"
```

Comment:
Why should the last two characters be newlines? I would love a comment here.

Reply:
This code turned out to be superfluous, I removed it.